feature/anotherfile.txt

```diff
@@ -0,0 +1,2 @@
+anotherfile
+new line in the another file.txt
```

Comment:
nice comment that you have decide to add this code

Reply:
thank you man!